components/crossplane-config/base/rbac.yaml

```diff
@@ -0,0 +1,35 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: crossplane-sa
+  namespace: crossplane-system
```

Comment:
The `crossplane-system` namespace won't exist on the EaaS clusters since we don't deploy crossplane there. I think you can put this SA and its RBAC in the `crossplane-config` namespace.